Doc/library/uuid.rst

```diff
@@ -19,8 +19,30 @@ If all you want is a unique ID, you should probably call :func:`uuid1` or
 a UUID containing the computer's network address.  :func:`uuid4` creates a
 random UUID.
 
+Depending on support from the underlying platform, :func:`uuid1` may or may
+not return a "safe" UUID.  A safe UUID is one which is generated using
+synchronization methods that ensure no two processes can obtain the same
+UUID.  All instances of :class:`UUID` have an :attr:`is_safe` attribute
+which relays any information about the UUID's safety, using this enumeration:
 
-.. class:: UUID(hex=None, bytes=None, bytes_le=None, fields=None, int=None, version=None)
+.. class:: SafeUUID
+
+.. attribute:: SafeUUID.safe
```

Comment:
Yeah, that looks nice, thanks.  The one problem is that `SafeUUID` still gets labeled as a class in the HTML.  Okay, technically it *is* a class, but it would be nice to have an `enum::` markup.  I grep'd around in the source and couldn't find any such examples, so I don't know if we support that yet.